src/endpoints/endpoint_registry.cpp

```diff
@@ -34,6 +34,14 @@ namespace ccf::endpoints
         ds::openapi::path(document, endpoint->full_uri_path),
         http_verb.value());
 
+      // Add what appears a *mandatory* operationId, which is expected to be
+      // unique across the spec.
+      std::string p =
+        std::regex_replace(endpoint->full_uri_path, std::regex("[/_]"), "");
+      std::string s = llhttp_method_name(http_verb.value());
+      ccf::nonstd::to_lower(s);
+      path_op["operationId"] = fmt::format("{}_{}", p, s);
```

Comment:
If the path is templated, it will contain `{` and `}`. Is that ok in an `operationId`?

The spec is gloriously vague about this:

> it is RECOMMENDED to follow common programming naming convention

Reply:
I've learned that nswag generates different C# stubs depending on how `operationId` is formatted.  For example, `_` causes multiple stubs to be generated.  Will investigate a bit more to find out what "common programming naming convections" are.

Reply:
The `{` and `}` are now eliminated.